@navikt/core/react/src/util/i18n/locales/nn.ts

```diff
@@ -37,4 +37,7 @@ export default {
       labelSuffix: "slett",
     },
   },
+  ErrorSummary: {
+    Heading: "Du må retta desse feila før du kan halda fram:",
```

Comment:
```suggestion
    Heading: "Du må rette desse feila før du kan halde fram:",
```
Although the a-suffix in "retta" and "halda" is technically not wrong, the e-suffix variant is more frequently used and familiar. (I discussed it with Magnus.)